packages/micropip/src/micropip/_micropip.py

```diff
@@ -365,7 +397,25 @@ def install(requirements: Union[str, List[str]], keep_going: bool = False):
     )
 
 
-__all__ = ["install"]
+def _list():
+    """Get the dictionary of installed packages through micropip.
```

Comment:
```suggestion
    """Get the dictionary of installed packages.
```
If we also support Pyodide packages here.